packages/databases-collections/src/components/collections/collections.jsx

```diff
@@ -118,6 +120,7 @@ function createEmit(evtName) {
 }
 
 const mapDispatchToProps = {
+  onRefreshClick: createEmit('refresh-data'),
```

Comment:
This will refresh all collections, databases, and data in any open collection tab, not just collections in a database list, is this really expected behavior?

Reply:
Oh no it is not. Actually the refresh of collection was not in original design but I added it as it made sense. Just got a go ahead from Claudia for this. I will modify this only to refresh the dataset for database.

Reply:
Same applies for databases, `refresh-data` refreshes a lot of things in the app. Do we want to have a more specific `refresh-databases` and `refresh-collections` (with a namespace) instead?

Reply:
Yes we do, this I addressed just now in recent commits.